scanpy/tests/test_plotting.py

```diff
@@ -1456,3 +1456,48 @@ def test_color_cycler(caplog):
             plt.close()
 
     assert caplog.text == ""
+
+
+@pytest.mark.parametrize(
+    "plot",
+    (
+        sc.pl.rank_genes_groups_dotplot,
+        sc.pl.rank_genes_groups_heatmap,
+        sc.pl.rank_genes_groups_matrixplot,
+        sc.pl.rank_genes_groups_stacked_violin,
+        sc.pl.rank_genes_groups_tracksplot,
+        # TODO: add other rank_genes_groups plots here once they work
+    ),
+)
+def test_filter_rank_genes_groups_plots(plot, check_same_image):
+    adata = sc.datasets.pbmc68k_reduced()
+
+    sc.tl.rank_genes_groups(adata, 'bulk_labels', method='wilcoxon', n_genes=5)
+
+    args = {
+        'adata': adata,
+        'key_added': 'rank_genes_groups_filtered',
+        'min_in_group_fraction': 0.25,
+        'min_fold_change': 1,
+        'max_out_group_fraction': 0.5,
+    }
+    sc.tl.filter_rank_genes_groups(**args)
```

Comment:
I don't think `args` is reused anywhere, so I think it this would be easier to follow without it

```suggestion
     sc.tl.filter_rank_genes_groups(
        adata,
        key_added='rank_genes_groups_filtered',
        min_in_group_fraction=0.25,
        min_fold_change=1,
        max_out_group_fraction=0.5,
     )
```